Systematics/python/flashggDiPhotonSystematics2018_cfi.py

```diff
@@ -386,7 +386,7 @@
 )
 
 
-scalesAndSmearingsPrefix = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2018_Step2Closure_CoarseEtaR9Gain")
+scalesAndSmearingsPrefix = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2018_Step2Closure_CoarseEtaR9Gain_v2")
```

Comment:
Changes in this file should not be part of this PR. Changes here can be reverted

Reply:
This is fine, this PR is dev on t&p and for that you need the latest corrections